v0/destinations/appcues/transform.js

```diff
@@ -0,0 +1,141 @@
+const { EventType } = require("../../../constants");
+
+const {
+  constructPayload,
+  defaultRequestConfig,
+  getFieldValueFromMessage,
+  getValueFromMessage,
+  removeUndefinedAndNullValues,
+  removeUndefinedAndNullAndEmptyValues
+} = require("../../util");
+
+const { ConfigCategory, mappingConfig, getEndpoint } = require("./config");
+
+function buildResponse(payload, endpoint) {
+  const response = defaultRequestConfig();
+  response.endpoint = endpoint;
+  response.body.JSON = removeUndefinedAndNullValues(payload);
+  return {
+    ...response,
+    headers: {
+      "Content-Type": "application/json"
+    }
+  };
+}
+
+function processTrackEvent(message, destination, mappingJson) {
+  const requestJson = {};
+  if (message.messageId) {
+    requestJson.request_id = message.messageId;
+  }
```

Comment:
Is there any issue mapping this using the mapping JSON?

Reply:
No issues, implemented that way now